src/lib/user-ip-address.js

```diff
@@ -0,0 +1,17 @@
+const net = require("net");
+
+module.exports = (request) => {
+  const forwarded = request.headers["forwarded"];
+  if (forwarded) {
+    const myObj = {};
+    const forwardedArray = forwarded.split(";");
+    for (const val of forwardedArray) {
+      const [key, value] = val.split("=");
+      myObj[key] = value;
+    }
+    const clientIp = myObj.for.toString().trim();
```

Comment:
if `for` didnt exist in the object, this would throw an error.

For example, imagine the following `forward`
`"host=subdomain.example.gov.uk;proto=https"`
or
`"for=;host=subdomain.example.gov.uk;proto=https"`

Then line 12 might throw an `cannot `access .toString() of undefined` error.

One solution would be to change the object accessor to `myObj?.for.toString().trim();`
This should return undefined into the clientIp